src/jsonLogic.js

```diff
@@ -125,6 +144,9 @@ function replaceHandlebarsTemplates({
   } else if (typeof toReplace === 'object') {
     const { value, ...rules } = toReplace;
 
+    if (Object.keys(rules).length > 1 && !value)
+      throw Error('Cannot define multiple rules without a template string with key `value`.');
+
```

Comment:
eslint: missing the brace `if { ... }` as we talked in the other PR

Reply:
since this is inline rules, its gone from here but will be reintroduced later down in that relevant PR